sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -4231,6 +4230,51 @@ object functions {
   // Scala UDF functions
   //////////////////////////////////////////////////////////////////////////////////////////////
 
+  /**
+   * Obtains a [[UserDefinedFunction]] that wraps the given [[Aggregator]]
```

Comment:
@erikerlandson, can you switch `[[UserDefinedFunction]]` to `` `UserDefinedFunction` ``? The documentation generation failure is due to this line.

```
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/functions.java:4326: error: reference not found
[error]    * Obtains a {@link UserDefinedFunction} that wraps the given {@link Aggregator}
[error]                       ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/functions.java:4326: error: reference not found
[error]    * Obtains a {@link UserDefinedFunction} that wraps the given {@link Aggregator}
[error]                                                                        ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/functions.java:4345: error: reference not found
[error]    * Obtains a {@link UserDefinedFunction} that wraps the given {@link Aggregator}
[error]                       ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder@2/sql/core/target/java/org/apache/spark/sql/functions.java:4345: error: reference not found
[error]    * Obtains a {@link UserDefinedFunction} that wraps the given {@link Aggregator}
[error]        
```

This failure is because we generate javadoc from scaladoc by Unidoc plugin. `UserDefinedFunction` seems `sealed abstract class` which is Scala specific keyword. In such cases, it uses different class names when it's reverse-compiled to Java classes by Unidoc. It's best to work around it just by wrapping it `` `...` `` for now.